### PR TITLE
ADM-613:[frontend]fix: fix the issue of Source control verify part wi…

### PR DESCRIPTION
**Pull request by Zxx-Ivy**
## Summary

Source control verify part without gitHub field

## Before

_Description_
image

## After

_Description_
image

### CODE REVIEW

File: frontend/__tests__/src/components/Metrics/MetricsStepper/MetricsStepper.test.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ import {
   updateDeploymentFrequencySettings,
   updateTreatFlagCardAsBlock,
 } from '@src/context/Metrics/metricsSlice'
-import { exportToJsonFile } from '@src/utils/util'
+import { exportToJsonFile, findCaseInsensitiveType } from '@src/utils/util'
```
Comment (guzhongren) [inline frontend/__tests__/src/components/Metrics/MetricsStepper/MetricsStepper.test.tsx:40]: See the GitHub Actions check in the PR, remove the unused variable

File: frontend/__tests__/src/utils/Util.test.tsx [hunk 2]
```diff
@@ -52,3 +58,36 @@ describe('getJiraToken function', () => {
     expect(jiraToken).toBe('')
   })
 })
+
+describe('findCaseInsensitiveType function', () => {
+  it('Should return as expected when passing in a type that gives case insensitive input bUildkite', () => {
```
Comment (guzhongren) [inline frontend/__tests__/src/utils/Util.test.tsx:63]: 1. follow team's test case pattern, given when then
2. what is `as exected`? PLZ update

File: frontend/__tests__/src/utils/Util.test.tsx [hunk 3]
```diff
@@ -52,3 +58,36 @@ describe('getJiraToken function', () => {
...
+    const value = findCaseInsensitiveType(Object.values(PIPELINE_TOOL_TYPES), selectedValue)
+    expect(value).not.toBe(PIPELINE_TOOL_TYPES.BUILD_KITE)
+    expect(value).not.toBe(PIPELINE_TOOL_TYPES.GO_CD)
+  })
+
+  it('Should return empty string when passing in a type that gives empty string ', () => {
+    const selectedValue = ''
+    const value = findCaseInsensitiveType(Object.values(PIPELINE_TOOL_TYPES), selectedValue)
+    expect(value).toBe(EMPTY_STRING)
+  })
```
Comment (guzhongren) [inline frontend/__tests__/src/utils/Util.test.tsx:92]: same issue

**Review by guzhongren — COMMENTED** [review]
Change with the comments

File: frontend/__tests__/src/utils/Util.test.tsx (hunk 2 above)
Comment (Zxx-Ivy) [reply frontend/__tests__/src/utils/Util.test.tsx:63]: Ok, will update later

File: frontend/__tests__/src/utils/Util.test.tsx (hunk 3 above)
Comment (Zxx-Ivy) [reply frontend/__tests__/src/utils/Util.test.tsx]: done

File: frontend/__tests__/src/utils/Util.test.tsx (hunk 2 above)
Comment (Zxx-Ivy) [reply frontend/__tests__/src/utils/Util.test.tsx]: done

File: frontend/__tests__/src/components/Metrics/MetricsStepper/MetricsStepper.test.tsx (hunk 1 above)
Comment (Zxx-Ivy) [reply frontend/__tests__/src/components/Metrics/MetricsStepper/MetricsStepper.test.tsx]: done

**Review by guzhongren — APPROVED** [review]

**Pull request merged**